s3tests/object_test.go

```diff
@@ -1456,37 +1455,40 @@ func (suite *S3Suite) TestMultipartUploadInvalidPart() {
 	}
 }
 
-func (suite *S3Suite) TestMultipartUploadNoSuchUpload() {
-
-	/*
-		Resource : object, method: get
-		Scenario : check failure on multiple multi-part upload with invalid upload id
-		Assertion: fails.
-	*/
-	assert := suite
-	bucket := GetBucketName()
-	num_parts := 2
-
-	payload := strings.Repeat("12345", 1024*1024)
-	key_name := "mymultipart"
+// func (suite *S3Suite) TestMultipartUploadNoSuchUpload() {
```

Comment:
it looks like this test is running after other multipart upload tests that use the same bucket and object name, and that's changing the behavior of this test. could you try with a different key_name for each?

Reply:
That does not seem to be the issue. I've tried using other key names, but the error remains the same.